src/algorithms.jl

```diff
@@ -281,6 +281,54 @@ function GMRES(;
     return GMRES(orth, maxiter, krylovdim, tol, verbosity)
 end
 
+"""
+LSMR(; orth = KrylovDefaults.orth,atol = KrylovDefaults.tol,btol = KrylovDefaults.tol,conlim = 1/KrylovDefaults.tol,
+    maxiter = KrylovDefaults.maxiter,krylovdim = KrylovDefaults.krylovdim,λ = 0.0,verbosity = 0)
```

Comment:
I think this docstring could use some formatting improvements and a bit of a clean up 😉